fdbkubernetesmonitor/kubernetes.go

```diff
@@ -309,6 +319,17 @@ func (podClient *PodClient) OnUpdate(_, newObj interface{}) {
 			return
 		}
 
+		// If the IsolateProcessGroupAnnotation changes force a reload of the configuration to make sure the processes
+		// will be shutdown.
+		previousIsolateProcessGroupAnnotationValue := previousAnnotations[IsolateProcessGroupAnnotation]
+		newIsolateProcessGroupAnnotationValue := podClient.podMetadata.Annotations[IsolateProcessGroupAnnotation]
+		if previousIsolateProcessGroupAnnotationValue != newIsolateProcessGroupAnnotationValue {
+			podClient.Logger.Info("Got change in isolate process group annotation", "previous", previousIsolateProcessGroupAnnotationValue, "new", newIsolateProcessGroupAnnotationValue)
+			podClient.TimestampFeed <- time.Now().Unix()
```

Comment:
We should update the description of `podClient.TimestampFeed` on the struct definition to something more like 
```
// TimestampFeed is a channel where the pod client will send updates with
// the values from OutdatedConfigMapAnnotation or IsolateProcessGroupAnnotation to prompt 
// configuration reloads
TimestampFeed chan int64
```